### DISCUSSION TITLE
How to convert a webp to gif formated

### DISCUSSION THREAD

**Discussion by ThisMe124** [ImageMagick/ImageMagick6#326] [General]
### ImageMagick version

ImageMagick 6.9.11-60 Q16 x86_64

### Operating system, version and so on

Pterodactyl Host

### Description

Here I want to ask about how to convert from webp to gif formated? 
I have tried here on terminal :

`convert sticker.webp sticker.gif` 

The Original Webp :

`https://qu.ax/LuiL.webp`

But after I tried it I got a gif like this :

`https://qu.ax/MoiJ.gif` 

How can I make the white line disappear while it's moving, I mean with clean results? 

Btw I'm still new to using imagemagick😐

**Comment by fmw42** [comment]
When you say your version is "latest" does that mean IM 7 or IM 6.  Please always give your exact version 7.x.x.x or 6.x.x.x

**Reply by fmw42** [reply]
Try the following assuming IM 7.  If not, change magick to convert.

`magick LuiL.webp -coalesce -layers optimize anim.gif`

![anim](https://github.com/ImageMagick/ImageMagick/assets/13458218/89a33ae2-2883-4b4c-b9f9-ee602321f7ca)

**Reply by ThisMe124** [reply]
> Try the following assuming IM 7. If not, change magick to convert.
> 
> `magick LuiL.webp -coalesce -layers optimize anim.gif`
> 
> ![anim](https://private-user-images.githubusercontent.com/13458218/339043783-89a33ae2-2883-4b4c-b9f9-ee602321f7ca.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyODc4OTcsIm5iZiI6MTcxODI4NzU5NywicGF0aCI6Ii8xMzQ1ODIxOC8zMzkwNDM3ODMtODlhMzNhZTItMjg4My00YjRjLWI5ZjktZWU2MDIzMjFmN2NhLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDE0MDYzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyZGJlNmNiOTM5YjhhNWY2ZTE4OWZmMzNhYTQ1ZDg0MmFiZmE0MjdmYTQ1ZjM5NDI0OTNhMWY3YjM2NmFmMmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oTD0EBfDt6KFI-s-DQ0abn7dtBSKS1egLP51uqamh-Y)

I used "ImageMagick 6.9.11-60 Q16 x86_64", and i use your give a `magick LuiL.webp -coalesce -layers optimize anim.gif` and same giving a white line like this `https://qu.ax/MoiJ.gif`

**Reply by fmw42** [reply]
On 6.9.11-60 use

`convert LuiL.webp -coalesce -layers optimize anim.gif`

However, when I do that on my IM 6.9.13-11 with libwebp 1.4.0, I do see the white outlines.  I think this is a bug, since it works in IM 7 for me as above.

![anim](https://github.com/ImageMagick/ImageMagick6/assets/13458218/6c6f3574-8a7a-40c0-a46e-9dacb96e27f9)



You should report this as a bug and reference this topic in the discussions.

**Reply by fmw42** [reply]
I have asked one of the developers to move this discussion to a bug report. So you do not need to do that unless he cannot move it.

**Reply by ThisMe124** [reply]
So would that be the bug? Then there is no other way to solve it because it is a bug?

**Reply by fmw42** [reply]
I believe it is a bug. But an IM developer will need to look into it.

**Reply by ThisMe124** [reply]
Uh, ok🙂

**Reply by ThisMe124** [reply]
> I have asked one of the developers to move this discussion to a bug report. So you do not need to do that unless he cannot move it.

https://github.com/ImageMagick/ImageMagick6/issues/327 someone has already reported it

**Reply by dlemstra** [reply]
This has been filed as an issue but it might take a while before we will have time to take a look at it.

**Reply by ThisMe124** [reply]
Okay...

**Reply by ThisMe124** [reply]
> I believe it is a bug. But an IM developer will need to look into it.

I found how to fix it, but to crop at the bottom. And i dont now how fix the crop🙃🙃

`convert -dispose previous -coalesce LuiL.webp result.gif`

The Result : `https://qu.ax/kWCA.gif`
